audio-worklet/style.css

```diff
@@ -43,33 +43,33 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 }
 
 div {
-	font: 100%/1.6 arial, georgia, serif;
+  font: 100%/1.6 arial, georgia, serif;
```

Comment:
Is the font really important?  Can't I use whatever font I have my browser set for?